Add training materials for non designers (#686)
etn-ccis/blui-doc-it: src/docs/design/for-non-designers.mdx

import { ImageGrid, LockedBehindEatonFirewall } from '../../app/components';
import ByNonDesigners from '../../app/assets/design/by-non-designers.webp';
import ByDesigners from '../../app/assets/design/by-designers.webp';

# Design Training for Non-designers

> Design is a critical part of product development, and there’s no substitute for having a dedicated designer embedded with the product team. That said, every team member can benefit from acquiring design skills — engineers can better understand the product benefit and design intent.

Reviewer: ```suggestion
Design is a critical part of product development, and there’s no substitute for having a dedicated designer embedded with the product team. That said, every team member (including engineers and managers) can benefit from acquiring design skills. Improving your understanding of design principles will improve your ability to communicate with designers, ask thoughtful questions about designs, understand design intent, and more.
```

<ImageGrid
    images={[ByNonDesigners, ByDesigners]}

>     caption={`A comparison of a Brightlayer UI application designed by different people. Can you guess which one is designed by a designer? Why do you guess so other than "it looks nice"?`}

Reviewer: ```suggestion
    caption={`Two designs of the same interface by individuals with different levels of design experience / knowledge. Learning more about design will improve the quality of your products, even if you don't have a dedicated designer.`}
```

/>

> Listed below are some recommended design courses for non-designers. Most of these courses are behind a paywall, but they offer a better learning experience compared to reading materials.

Reviewer: ```suggestion
Listed below are some recommended design courses for non-designers. Most of these courses require a paid subscription, but the best content often does. These courses offer a much more robust learning experience compared to free materials / articles that you might find elsewhere.
```

# User Experience Design

> User experience (UX) design involves a broad spectrum of expertise, from market research to design thinking. It is a critical skill for anyone on a front-end team who is expected to conceptualize new features from scratch.

Reviewer: ```suggestion
User experience (UX) design covers a broad spectrum of topics, from market research to design thinking. The key focus here is understanding the problem you are trying to solve and ensuring that you solve it in the right way. It is a critical skill for anyone on a front-end team, especially if you will be responsible for envisioning new concepts / features.
```

The UX training is better conducted in person with an experienced instructor. You may also rely on online resources to help you deepen your understanding of the domain.

<p style={{ display: 'flex', alignItems: 'center', gap: 8 }}>
    <strong>Eaton Design Thinking Training</strong>
    <LockedBehindEatonFirewall />
</p>
{/* <!-- markdown-link-check-disable --> */}

> -   [Our Training Program - Design Thinking Resources](<https://eaton.sharepoint.com/sites/ERG_DesignThinkingPractice/SitePages/Training(1).aspx>)

Reviewer: ```suggestion
-   [Studio Blue Training Program - Design Thinking Resources](<https://eaton.sharepoint.com/sites/ERG_DesignThinkingPractice/SitePages/Training(1).aspx>)
```

{/* <!-- markdown-link-check-enable --> */}
**LinkedIn Learning**

-   [User Experience (UX) for Non-Designers](https://www.linkedin.com/learning/user-experience-ux-for-non-designers)

# User Interface Design

> The user interface (UI) design deals with visual designs — the most efficient way to present a piece of information. It is a good skill to have for team members who need to improvise to incorporate a small feature into an established UI.

Reviewer: ```suggestion
User interface (UI) design deals with the visual side of design — how will you best present a piece of information on the screen. It is a good skill to have for team members who may need to improvise or incorporate small features into an established UI.
```
